repose-aggregator/pom.xml

```diff
@@ -31,6 +31,8 @@
         <jackson.version>1.9.13</jackson.version>
         <yammer.version>2.2.0</yammer.version>
         <jetty.version>9.2.0.v20140526</jetty.version>
+        <!-- NOTE: the saxon version should match that of API-checker, because there is saxon in core :( -->
+        <saxon.version>9.4.0.9</saxon.version>
```

Comment:
We might only have saxon in one other place... In the translation filter, which uses both saxon dependencies. I'd like to keep this, since we manage many other versions.